IBAnalyzer/Analyzers/ConnectionAnalyzer.swift

```diff
@@ -7,25 +7,127 @@
 //
 
 import Foundation
+import SourceKittenFramework
+
+struct Violation {
```

Comment:
I like how this structure encapsulates some additional data needed for better warning/error reporting but I think its name should be improved. At the time of parsing we only know that it represents an action or outlet but we aren't yet sure if it'll be a violation.

I'd consider naming it `Declaration` instead.